osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs

```diff
@@ -357,19 +359,23 @@ private double calculateComboBasedEstimatedMissCount(OsuDifficultyAttributes att
 
         private double calculateEstimatedSliderBreaks(double topWeightedSliderFactor, OsuDifficultyAttributes attributes)
         {
-            if (!usingClassicSliderAccuracy || countOk == 0)
+            // Only ok and meh can be a sliderbreak
+            int okMehs = countOk + countMeh;
```

Comment:
While that makes sense I'm not sure if we should do it like that exactly. Sliderbreaks that give you a 50 are way less common than ones that give you a 100 and I feel like we might start grossly overestimating on scores with lots of 50s (which almost always come from mistapping and not sliderbreaking). I'm not sure what solution would be the best here - lowering 50 influence or just not accounting for them at all because of its rarity.

@tsunyoku will need you onion

Reply:
I think we can just remove the inclusion of 50s for now. We can revisit at a later point if it poses an issue, but my main goal with this PR was to fix the limits so I really don't want this PR to turn into a debate for what should be a simple fix.

@Givikap120 if you feel strongly about including 50s, or maybe want to include them with a lesser weight, please do it as a separate PR so that this fix can remain simple.

Reply:
Considering that 50s are usually LTMs, not sliderhead misses, and this algorithm mainly targets sliderhead misses as LTMs - I can agree with this part being removed.